make-utils.jl

```diff
@@ -310,6 +312,8 @@ function copy_tutorial(tutorial_dest_path)
     finally
         # Clean up temporary workspace
         rm(tmp_path, recursive=true)
+
+        rm(OUTPUT_DIR, recursive=true)
```

Comment:
Isn't
```suggestion
        rm(md_path, recursive=true)
```
better, in case one wants to rerun the function and/or avoid re-cloning the repo?

Reply:
Probably, but it's very hard to keep the repo in the right updated state so that's why I probably did it like this. As long as you don't restart Julia, it won't run again.

Reply:
The Github action restarts Julia (runs the script twice for different versions of Turing), so I think it's preferable to not reclone the repo. Maybe it's good though to do a clone or update above, i.e., to update to the latest commit if the TuringTutorialsOutput repo is already checked out? Or is this already implied by `clone`? It does not matter for the action here but I guess it would be more convenient if you run the script locally.

Reply:
Could you finish the PR yourself? That is a much more efficient way than this back and forth messaging with hours in between.

Reply:
Or let's just merge and fix that nitpick later. The only thing in the way of a good PR is the dream of a perfect PR.

Reply:
Well, actually I don't dare to finish, approve, and/or merge this PR. The main reason is that I'm not familiar with all workarounds, why they were added, and why they are required (or why they could be removed). Even more so since I almost missed that removing the deploy config and changing the action would have broken builds on the master branch. So I think this PR needs @cpfiffer's approval before merging it.

While looking at the action once more, I just noticed that in fact Julia is restarted and the repo will be recloned - which your comment above seemed to suggest would be the only case when removing it would be undesirable.

Reply:
To be honest, I'm not going to be of much use here -- I'm in dissertation mode and I can't dedicate the time required to dig back into this stuff and figure out all the tape and glue that makes this thing work. 

If @rikhuijzer is fine with taking point here, I say we let him push it through and make changes as needed.

Reply:
I second @cpfiffer's suggestion here. It is not a big deal breaking some stuff here (everything is tracked and can be fixed with additional PR), particularly the docs building is already broken. 

@rikhuijzer would you be happy to push this through?